backend/src/main/java/com/huellapositiva/infrastructure/SecurityConfig.java

```diff
@@ -23,12 +25,25 @@ PasswordEncoder passwordEncoder() {
         return new BCryptPasswordEncoder();
     }
 
+    @Bean
+    public WebMvcConfigurer corsConfigure(){
+        return new WebMvcConfigurer() {
+            @Override
+            public void addCorsMappings(CorsRegistry registry) {
+                registry.addMapping("/api/v1/**")
+                        .allowedOrigins("http://localhost:3000")
```

Comment:
Externalize this value to a property that can be set in the different environments or it will not work in DEV and PROD

Reply:
Okay, done.